xedocs/schemas/corrections/implementations/se_gain_partition.py

```diff
@@ -0,0 +1,22 @@
+"""
+Correction: Region linear and circular for AB/Cd partitions
```

Comment:
```suggestion
Correction: Region linear and circular for AB/CD partitions
```

Sorry for being pedantic... 🙈